airbyte-webapp/src/pages/ConnectorBuilderPage/ConnectorBuilderPage.tsx

```diff
@@ -29,16 +29,25 @@ const ConnectorBuilderPageInner: React.FC = React.memo(() => {
   const initialFormValues = useRef(builderFormValues);
   return useMemo(
     () => (
-      <Formik initialValues={initialFormValues.current} onSubmit={noop} validationSchema={builderFormValidationSchema}>
-        {({ values, validateForm }) => (
-          <Panels
-            editorView={editorView}
-            validateForm={validateForm}
-            switchToUI={switchToUI}
-            values={values}
-            switchToYaml={switchToYaml}
-          />
-        )}
+      <Formik
+        initialValues={initialFormValues.current}
+        validateOnBlur={false}
```

Comment:
Noticed that formik would still re-trigger renders if all validateOn* are explicitly disabled - this is a big saving.